session.go

```diff
@@ -579,6 +579,10 @@ func (s *session) sendPacket() error {
 			utils.Debugf("\tDequeueing retransmission for packet 0x%x", retransmitPacket.PacketNumber)
 
 			if retransmitPacket.EncryptionLevel != protocol.EncryptionForwardSecure {
+				if s.handshakeComplete {
+					// Don't retransmit handshake packets when the handshake is complete
+					continue
+				}
 				utils.Debugf("\tDequeueing handshake retransmission for packet 0x%x", retransmitPacket.PacketNumber)
```

Comment:
True, but I didn't want to touch the line (to not cause merge conflicts for your log work).